cmd/bigsky/main.go

```diff
@@ -17,9 +17,11 @@ import (
 	"github.com/bluesky-social/indigo/notifs"
 	"github.com/bluesky-social/indigo/plc"
 	"github.com/bluesky-social/indigo/repomgr"
+	"go.opencensus.io/trace"
 
 	_ "net/http/pprof"
 
+	datadog "github.com/DataDog/opencensus-go-exporter-datadog"
```

Comment:
This repo says:

> This repository is deprecated and is in maintenance mode.

But it does also say that bugs are getting fixed, and the alternative linked to seems more complex and "beta" status. Did you chose this one intentionally?